.github/workflows/gfmrun.yml

```diff
@@ -91,5 +91,5 @@ jobs:
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
           asset_path: ./gfmrun-windows-amd64-${{ steps.extract_version.outputs.VERSION }}.exe
-          asset_name: gfmrun_win.exe
+          asset_name: gfmrun-win-${{ steps.extract_version.outputs.VERISON }}.exe
```

Comment:
Missing arch and should be windows to match goos name

Reply:
Yes. I noticed it and I've pushed a fix